.jenkins

```diff
@@ -314,8 +314,67 @@ pipeline {
                         }
                     }
                 }
+
+                stage('HIP-3.5') {
+                    agent {
+                        dockerfile {
+                            filename "Dockerfile.hipcc"
+                            dir "docker"
+                            additionalBuildArgs '--build-arg BASE=rocm/dev-ubuntu-18.04:3.5'
+                            args '-v /tmp/ccache.kokkos:/tmp/ccache --device=/dev/kfd --device=/dev/dri --security-opt seccomp=unconfined --group-add video'
+                            label 'rocm-docker && vega'
+                        }
+                    }
+                    steps {
+                        sh 'ccache --zero-stats'
```

Comment:
Remind me did we have issues with ccache?

Reply:
Yes, we had (both in `Kokkos` and here). Do you want me to remove everything `ccache` related here?